### PR TITLE
The same contract performs selfdestruct twice (part 2/3)

### PR DESCRIPTION
**Pull request by pirapira**
This adds the second case requested in https://github.com/ethereum/tests/issues/431

### CODE REVIEW

File: src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTest2Filler.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+doubleSelfdestructTest2:
+  info:
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTest2Filler.yml]: _info

Comment (pirapira) [reply src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTest2Filler.yml]: I'll fix.

Comment (pirapira) [reply src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTest2Filler.yml]: Suddenly I realize that the schema checking doesn't work on YAML.  I created an issue https://github.com/ethereum/tests/issues/456

**Comment by pirapira** [comment]
@winsvega, I fixed the wrong field name.

File: GeneralStateTests/stSystemOperationsTest/doubleSelfdestructTest2.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
+{
+    "doubleSelfdestructTest2" : {
+        "_info" : {
+            "comment" : "The second test case requested here https://github.com/ethereum/tests/issues/431#issue-306081539 .\nInvoked Solidity 0.4.21 with solc --bin-runtime --optimize\ncontract DoubleSelfdestructTest2 {\n  address constant B = 0x3;\n  function remoteSelfdestruct() external {\n    selfdestruct(this);\n  }\n  function() external payable {\n    this.remoteSelfdestruct();\n    selfdestruct(B);\n  }\n}\nThe version of Solidity emits code that always enter OOG for Homestead and Frontier.  So no new accounts\nare created for Homestead and Frontier.  EIP158 changes the behavior of zero-amount transfer so that no new accounts are created.\nThis leaves EIP150 as a special case.\n",
```
Comment (winsvega) [inline GeneralStateTests/stSystemOperationsTest/doubleSelfdestructTest2.json:4]: I see. now this source does not look readable in the final test. to solve this issue, I usually put code source only in the test filler in the //comment field

**Pull request merged**